UndertaleModLib/Util/GMImage.cs

```diff
@@ -452,6 +470,31 @@ public static GMImage FromQoi(byte[] data)
         return new GMImage(ImageFormat.Qoi, width, height, data);
     }
 
+    /// <summary>
+    /// Creates a <see cref="GMImage"/> of DDS format, wrapping around the provided byte array containing DDS data.
+    /// </summary>
+    /// <param name="data">Byte array of DDS data.</param>
+    public static GMImage FromDds(byte[] data)
+    {
+        ArgumentNullException.ThrowIfNull(data);
+
+        ReadOnlySpan<byte> span = data.AsSpan();
+
+        // Get height and width
+        int height = (int)BinaryPrimitives.ReadUInt32LittleEndian(span[12..16]);
+        int width = (int)BinaryPrimitives.ReadUInt32LittleEndian(span[16..20]);
+
+        // Create wrapper image
+        return new GMImage(ImageFormat.Dds, width, height, data);
+    }
+
+    private void AddMagickToPngSettings(MagickReadSettings settings)
```

Comment:
Unsure if this is going to stick around in the long run, but this can probably be marked `static`.

Reply:
Totally